tests/core_tests/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ set(core_tests_sources
   wallet_tools.cpp)
 
 set(core_tests_headers
-  block_reward.h
```

Comment:
I don't think these need to be commented out, chaingen_main `#if 0`s the broken test cases out

Reply:
Is there a point in compiling code that isn't going to be run?

Reply:
It's disabling at 2 places which is more inconvenient than just setting 1 place for when someone needs to fix it.

Reply:
Okay, I'll reenable.  (This was were I started adding tests until I realized it wasn't running at all -- then I tried to fix the tests, and that... didn't work out so well).